frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -861,7 +861,11 @@ end
 
 function ReaderPaging:onScrollPageRel(page_diff)
     if page_diff == 0 then return true end
-    if page_diff > 0 then
+    if page_diff > 1 then
+        -- More than 1 page, don't bother with how far we're scrolled.
```

Comment:
Any reason to not do this also below when page_diff < -1 ?

Reply:
Whoops, you're totally right.

Reply:
That's what happens when you quickly whip up something in five minutes, but that's also why we have the review process. :+1: